src/docs/product/releases/health/index.mdx

```diff
@@ -26,21 +26,29 @@ A session ends with the closing of the application or with the application being
 
 Sessions are submitted to Sentry so you can track the usage and adoption of your application. When a user of your application experiences a crash, error, or abnormal exit, the session will be flagged accordingly, and Sentry calculates derived metrics. The metrics include data such as the number of users that didn't experience a crash in the specified time range.
 
-### Active Users
+### Active Sessions / Users
 
-Users who started the application at least once in the last 24 hours.
+Active Sessions are sessions that started in the application in the last 24 hours.
+
+Active Users are users who started the application at least once in the last 24 hours.
+
+To sort releases by active sessions, select Display: Sessions and to sort by active users select Display: Users.
+
+![Sort By: Active Sessions / Users dropdown](release_active.png)
```

Comment:
This screenshot is a confusing. Recommend you split them into two and place one after the other.

Reply:
👍